lang/py/avro/errors.py

```diff
@@ -83,3 +83,7 @@ class UnsupportedCodec(NotImplementedError, AvroException):
 
 class UsageError(RuntimeError, AvroException):
     """An exception raised when incorrect arguments were passed."""
+
+
+class AvroRuntimeException(AvroException):
```

Comment:
Can this be a subclass of both `RuntimeError` and `AvroException` like `UsageError` above?

Reply:
fixed